website_signup_legal_page_required/__manifest__.py

```diff
@@ -9,8 +9,7 @@
     "version": "12.0.1.0.0",
     "category": "Website",
     "website": "https://github.com/OCA/website",
-    "author": "Antiun Ingeniería S.L.,"
-    " Agile Business Group, José Luis Sandoval Alaguna, Odoo Community Association (OCA)",
+    "author": "José Luis Sandoval Alaguna, Odoo Community Association (OCA)",
```

Comment:
Do not remove the original authors

Reply:
Even more, this migration doesn't grant you as author. Yous should put yourself in contributors, but not in authors. And please respect the attribution principle in open source not removing previous authors.